### DISCUSSION TITLE
Scenario "Bilbao": Basic Kubernetes Problems

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#49] [Scenarios]
Discussions about scenario [bilbao](https://github.com/fduran/sadservers/tree/main/scenarios/bilbao)

**Comment by sabir69261** [comment]
One of the nodes in this scenario is in a NotReady state. Also, node1 has limited resources, so the pod will not come online unless we reduce the memory configuration in manifest.yaml.

Please either fix the node issue or update manifest.yaml to use minimal resource requests

**Comment by fduran** [comment]
fixed
